nms/app/components/GatewayCommandFields.tsx

```diff
@@ -379,6 +381,15 @@ export function GenericCommandControls(props: ChildProps) {
     <div>
       <Divider className={classes.divider} />
       <Text variant="subtitle1">Generic</Text>
+      <br />
+      <Text variant="body2">
+        Allowed commands are configured in `magmad.yml`. Specify parameters in
+        JSON via "shell_params", i.e. `
+        {`
+        {"shell_params": ["param_1 ... param_n"]}
```

Comment:
Nit:
```suggestion
        {"shell_params": ["param_1", ... "param_n"]}
```

Reply:
That was actually on purpose. I also think it's not intuitive, but if you want to provide multiple parameters, I think you need to do it like ["param1 param2 param3"], not ["param1", "param2", "param3"]. I think this is something that we could suggest as an improvement in the issue to follow.

Reply:
You're right, looks like the other parameters are just ignored.